anda/tools/nerdfetch/anda.hcl

```
@@ -0,0 +1,5 @@
+project pkg {
+  rpm {
```

Funny CI thing, it'll always build for aarch64 *and* x86_64 unless specified otherwise, so for noarch packages to only build them once what we do is:
```suggestion
       arches = ["x86_64"]
  rpm {
```